internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1869,7 +1869,6 @@ components:
     EnterpriseOsdClusterPayloadExample:
       value:
         cluster_id: 1234abcd1234abcd1234abcd1234abcd
-        cluster_external_id: 69d631de-9b7f-4bc2-bf4f-4d3295a7b25e
         cluster_ingress_dns_name: apps.enterprise-aws.awdk.s1.devshift.org
```

Comment:
can we get that from ocm too?

Reply:
Yes, we can get the cluster dns. But the current logic in `GetClusterDNS` doesn't handle for the private case.
The CLI already does that and is providing us with the right value. We could keep it for now.
There is also future planned work where we might not need to provide this value at all as KFM won't need it: we could use that opportunity to get rid of cluster dns.

Reply:
Ok, we can handle that in a separate PR (the private cluster case) and let the cli handle it for the moment.

Reply:
@ziccardi that's more or less what we are doing at the moment with the `GetClusterDNS` method. But that doesn't work if the cluster is a private cluster - in this case the `ingresses` resources returns an empty list.